src/libs/NextStepUtils.ts

```diff
@@ -38,14 +38,16 @@ Onyx.connect({
 function parseMessage(messages: Message[] | undefined) {
     let nextStepHTML = '';
 
-    messages?.forEach((part) => {
+    messages?.forEach((part, index) => {
         const isEmail = Str.isValidEmail(part.text);
         let tagType = part.type ?? 'span';
         let content = Str.safeEscape(part.text);
 
         if (currentUserEmail === part.text || part.clickToCopyText === currentUserEmail) {
             tagType = 'strong';
-            content = 'You';
+            content = messages[index + 1].text === `'s` ? 'Your' : 'You';
+        } else if (part.text === `'s` && (messages[index - 1].text === currentUserEmail || messages[index - 1].clickToCopyText === currentUserEmail)) {
```

Comment:
Maybe add `const previousPart = messages[index - 1];` before the `if-else if-else`. And then here, in the `else if` check also that `previousPart !== undefined` so we don't risk a crash if you find `'s` in the first position of `messages`